app/containers/Questions/Content/Content.jsx

```diff
@@ -224,6 +229,32 @@ const QuestionItem = connect(
   }),
 )(QI);
 
+const Span = styled.span`
+  width: 100%;
+  background: ${PEER_PRIMARY_TRANSPARENT_COLOR};
+`;
+
+const Highlight = ({ filter, str }) => {
```

Comment:
the component is difficult to write, it can be simpler

move the component to a separate file, write the types of props

```
const Highlight = ({ filter, str }) => {
  const textArray = str.split(filter);

  return (
    <>
      {textArray.map((item, index) => (
        <Fragment key={item}>
          {item}
          {index !== textArray.length - 1 && <Span>{filter}</Span>}
        </Fragment>
      ))}
    </>
  );
};
```

Reply:
Done